merkletree/proof_test.go

```diff
@@ -142,15 +140,25 @@ func TestVerifyProof(t *testing.T) {
 	absentIndex := vrf.Compute([]byte("123"), vrfPrivKey1)
```

Comment:
L134-137 above are redundant now that `authPathHash` is moved inside `verifyProof`.